src/expressServer.js

```diff
@@ -100,7 +98,7 @@ app.set('view engine', 'ejs');
 app.use(favicon(__dirname + '/public/favicon.ico'));
 // Cookie parser middleware with JWT secret
 app.use(cookieParser(process.env.JWT_SECRET));
-
+app.use(express.static('public'));
```

Comment:
On line 139 there is already a line that says `app.use(express.static(`_**`__dirname`**_ `+ '/public'));`. That means that Express will always look in this path, `<the path where you have this backend repo stored>/src/public`, to find any files matching the current URL, _before_ it looks to match the current URL against any defined routes that come after line 139.

So for example if your browser requests `localhost:8000/my_special_file.js` , Express will look for a file in `<path to repo>/src/public/my_special_file.js` and return that to the browser. And this applies to anything after the `localhost:8000` part, not just Javascript files. 

But here on line 101 you've added a separate call to tell Express to also look for files in the `public` folder _not_ inside `src`. So now it's looking in two different places for arbitrary resources, which I think will cause confusion in the future and make things more difficult to debug.

Reply:
Removed separate call to public commit  38f8ea507d8892ec277bdd098227947c2d237b3c